salt/returners/redis_return.py

```diff
@@ -148,6 +150,7 @@ def _get_options(ret=None):
              'port': 'port',
              'unix_socket_path': 'unix_socket_path',
              'db': 'db',
+             'password': 'password',
```

Comment:
Should this be set to `''`?

Reply:
Not sure what you mean... this is in line with the other options here? I was surprised to see a default value of '' as well though (see question above) but I aligned it with the redis cache module for now to keep it consistent.